tests/console/apache.pm

```diff
@@ -35,6 +35,9 @@ sub run {
     # Even before the installation there should be htdocs so we can create the index
     assert_script_run 'echo "index" > /srv/www/htdocs/index.html';
 
+    # Change SELinux policy for port 85, poo#178240
```

Comment:
Do we want the `poo#178240` reference here?

Reply:
IMO not necessary but can't hurt either.